src/options/opt.cc

```diff
@@ -271,21 +273,21 @@ LOCAL_NODISCARD(Ret fix_mutopt(const conopt_t& glob,
         // In Go `continue` statements have labels, use it to avoid ambiguity.
         if (is_default.label_loop) real.label_loop = "yyl";
     }
-
     // errors
+    if (real.case_ranges && !(glob.lang == Lang::C || glob.lang == Lang::D)){
+    	RET_FAIL(error("--case-ranges option is supported only for C and D backends"));
+    }
+    if(real.api == Api::DEFAULT && !(glob.lang == Lang::C || glob.lang == Lang::D)){
+    	RET_FAIL(error("pointer API is supported only for C and D backends"));
+    }
+
```

Comment:
Since we have two cases, can you factor out `if !(glob.lang == Lang::C || glob.lang == Lang::D)) {` (add space before `if` and curly brace) and move it after the `if (glob.lang != Lang::C) {` block. Also, remove newline that is currently on line 283.